src/tools/time_governor.cc

```diff
@@ -48,8 +49,9 @@ const Tuple & TimeGovernor::get_input_time_type(double lower_bound, double upper
     return Tuple("TimeValue", "A time with optional unit specification.")
         .declare_key("time", Double(lower_bound, upper_bound), Default::obligatory(),
                                     "The time value." )
-		.declare_key("unit", String(), Default::read_time("Common time unit of the equation's Time Governor."),
-				"The time unit. Possible values: 's' seconds, 'min' minutes, 'h' hours, 'd' days, 'y' years.")
+		.declare_key("unit", UnitConverter::get_input_type(),
+                Default::read_time("The time unit."),
+				"User can benefit from the Unit Convertor funcionality, e.g. creating own time units.")
```

Comment:
Have to check all places accessing "time" and "unit" and call `UnitConverter::convert("s")`.

Reply:
Seems that TimeUnitConversion is the right place.